src/main/java/ui/UserInterface.java

```diff
@@ -149,13 +149,19 @@ private int getIntegerInput(String message) {
 	 * Method to get user inputs in form of a double
 	 * 
 	 * @param message prompt to display to user
-	 * @return user inputted double
+	 * @return user input double
 	 */
 	private double getDoubleInput(String message) {
 		do {
 			try {
 				String rawUserInput = getFirstWord(message);
-				return Double.parseDouble(rawUserInput);
+				Double userDoubleInput = Double.parseDouble(rawUserInput);
+				if(userDoubleInput.doubleValue() < 0) {
+					System.out.println("Input must be a positive number!");
+				}
+				else {
```

Comment:
"else" should be on same line as "}"

Reply:
I will be able to get that change made at 4:25.  I apologize I'm on a work computer without an IDE, I'll do it asap.